src/pages/proposal-commons/proposal-details/vote-form/vote-form.tsx

```diff
@@ -25,14 +25,14 @@ const VoteForm = (props: Props) => {
             label={<h4>For</h4>}
             onChange={() => setChecked('for')}
             checked={isVotingFor}
-            name="vote"
+            name="for"
```

Comment:
This is pretty annoying and for no good reason... I think I will remove this param from props and generate unique name inside the RadioButton component later.